SFTPStream.md

```diff
@@ -7,32 +7,54 @@ SFTPStream events
 
 **Server-only events**
 
+In all cases, responses to the various events **must** include the request's `reqID` as
+their first parameter. The `sftpStream` that the event fired on is the same object 
+that the various response methods should be invoked using.
+
 * **OPEN**(< _integer_ >reqID, < _string_ >filename, < _integer_ >flags, < _ATTRS_>attrs)
+ - respond with `sftpStream.handle()` with your newly-opened file handle. The file handle
+   should uniquely identify this opened file for this connection.
 
 * **READ**(< _integer_ >reqID, < _Buffer_ >handle, < _integer_ >offset, < _integer_ >length)
+ - respond with `sftpStream.data()` with the requested data. Respond with 
+   `sftpStream.status(ssh2.SFTP_STATUS_CODES.EOF)` if there is no more data to return.
```

Comment:
Avoid references to `ssh2`, since it technically is a separate project. If you need to reference specific values, use a format like `SFTPStream.STATUS_CODE.EOF`.